src/scrabble.js

```diff
@@ -1,5 +1,146 @@
-function scrabble() {
+const doubleArray = []
+const tripleArray = []
+const alphabet = [
+  'A',
+  'B',
+  'C',
+  'D',
+  'E',
+  'F',
+  'G',
+  'H',
+  'I',
+  'J',
+  'K',
+  'L',
+  'M',
+  'N',
+  'O',
+  'P',
+  'Q',
+  'R',
+  'S',
+  'T',
+  'U',
+  'V',
+  'W',
+  'X',
+  'Y',
+  'Z'
+]
 
+function scrabble(word) {
+  let score = 0
+  const check = wordCheck(word) // this function checks that the word only has acceptable characters
+  console.log(check) // this displays whether the word is valid
+  if (check === true) {
+    word = word.toUpperCase()
+    for (let i = 0; i < word.length; i++) {
+      const letter = word[i]
+      // console.log(letter) // this shows the individual letters been checked
+      score += scoring(letter)
+    }
+    if (doubleArray.length !== 0) {
+      console.log(doubleArray)
+      for (let i = 0; i < doubleArray.length; i++) {
+        const letter = doubleArray[i].toUpperCase()
+        score += scoring(letter) * 2
+      }
+    }
+    if (tripleArray.length !== 0) {
+      console.log(tripleArray)
+      for (let i = 0; i < tripleArray.length; i++) {
+        const letter = tripleArray[i].toUpperCase()
+        score += scoring(letter) * 3
+      }
+    }
+  }
+  return score
```

Comment:
I can see what you're trying to do here, I went for the unorthodox method of pushing the value surrounded by brackets onto the end of an array I created for the word (twice if it was square brackets), thus duplicating it so that it recieved a double score or triple score respectively.